resources/config.rb

```diff
@@ -24,10 +24,12 @@
          coerce: proc { |m| m.is_a?(Hash) ? Mash.new(m) : m }
 property :service_group, String, name_property: true, desired_state: false
 property :remote_sup, String, default: '127.0.0.1:9632', desired_state: false
+# Http port needed for querying/comparing current config value
+property :remote_sup_http, String, default: '127.0.0.1:9631', desired_state: false
```

Comment:
Not that it needs to be done here, but at some point we should get some validation on the input here